packages/mobile/src/paymentRequest/PaymentRequestConfirmation.tsx

```diff
@@ -117,7 +117,7 @@ class PaymentRequestConfirmation extends React.Component<Props> {
       timestamp: new Date(),
       requesterAddress: address,
```

Comment:
Should this address be normalized too?

Reply:
It's already doing that on the selector:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/web3/selectors.ts#L3
```
export const currentAccountSelector = (state: RootState) =>
  (state.web3.account && state.web3.account.toLowerCase()) || null
```